### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by compSciKai**
edited the README to increase formal language. Reduced use of pronouns to increase readability.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,22 +10,22 @@ A list of popular sites and whether or not they accept two factor auth.
...
-If you'd like to contribute, read the entire guidelines here in
+If you would like to contribute, please read the entire guidelines here in
 [CONTRIBUTING.md][contrib].
 
 ## Running Locally
 
 TwoFactorAuth.org is built upon [Jekyll](https://jekyllrb.com/), using the [github-pages](https://github.com/github/pages-gem) gem.
-In order to run the site locally, it is necessary to install bundler, install all dependencies, and then use Jekyll to serve
-the site. If the `gem` command is not available to you, it is necessary to install Ruby with RubyGems.
+In order to run the site locally, bundler, and all other dependencies will need to be installed, and afterwards Jekyll can be serve
```
Comment (phallobst) [inline README.md]: That should either be "Jekyll can serve" or "Jekyll can be used to serve".

**Review by phallobst — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (compSciKai) [reply README.md]: oops. I have fixed it :)

**Review by kmpoppe — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Comment by compSciKai** [comment]
Thank you for letting me work on your project. Have a great day!

**Comment by Carlgo11** [comment]
I don’t think we've ever had someone thanking us for letting them contribute to the project.

Thank you for thanking us 😄

**Pull request merged**